src/test/java/org/llorllale/cactoos/matchers/RunsInThreadsTest.java

```diff
@@ -75,18 +73,18 @@ public void matchNegative() {
         final AtomicInteger counter = new AtomicInteger(0);
         final int threads = 20;
         final int attempts = 100;
-        new Assertion<>(
-            "must not match Func that is not thread-safe",
-            () -> new RunsInThreads<>(counter, threads),
+        new Assertion2<>(
+            "does not match Func that is not thread-safe",
+            new RunsInThreads<>(counter, threads),
             new IsNot<>(
                 new Matches<>(
                     new RepeatedFunc<>(new Unsafe(), attempts)
                 )
             )
         ).affirm();
-        new Assertion<>(
+        new Assertion2<>(
             "counter must not be incremented by all threads",
-            counter::get,
+            counter.get(),
             new IsNot<>(new IsEqual<>(threads * attempts))
         ).affirm();
     }
```

Comment:
Exception 'java.lang.Exception' is never thrown in the methods here, why do we have `throws Exception` signature here?

Reply:
@ilyakharlamov I think you got confused here - there is no `throws Exception` in this test method

Reply:
@llorllale I meant `public Boolean apply(final AtomicInteger input) throws Exception {` no need for Exception here

Reply:
@ilyakharlamov fixed